### ISSUE TITLE
DL4006 triggered when using SHELL ["/bin/sh", "-ex", "-o", "pipefail", "-c"]

### ISSUE THREAD

**Issue by 4x0v7** [hadolint/hadolint#350]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
Rule is not triggered

### Actual behavior
Rule is triggered

### Steps to reproduce the behavior
Run hadolint on the below Dockerfile.
The rule doesn't trigger when using `/bin/ash` as [per the example](https://github.com/hadolint/hadolint/wiki/DL4006)

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.17.2-0-g574b169
```

**Dockerfile** (if relevant)

```Dockerfile
FROM alpine:3.10

SHELL ["/bin/sh", "-ex", "-o", "pipefail", "-c"]
RUN curl file | command -
```

**Additional environment details (OS, stack version, etc.)**

**Comment by lorenzo** [comment]
but sh does no support pipefail. I think the linter is right in this case

**Comment by 4x0v7** [comment]
It looks like `/bin/sh` is perhaps an alias of `ash` on Alpine

```
/ # sh --help
BusyBox v1.30.1 (2019-06-12 17:51:55 UTC) multi-call binary.

Usage: sh [-/+OPTIONS] [-/+o OPT]... [-c 'SCRIPT' [ARG0 [ARGS]] / FILE [ARGS] / -s [ARGS]]

Unix shell interpreter
/ # ash --help
BusyBox v1.30.1 (2019-06-12 17:51:55 UTC) multi-call binary.

Usage: ash [-/+OPTIONS] [-/+o OPT]... [-c 'SCRIPT' [ARG0 [ARGS]] / FILE [ARGS] / -s [ARGS]]

Unix shell interpreter
```

**Comment by lorenzo** [comment]
Ok, un that case, this is a duplicate of #250 

Please look at the suggestions in that ticket.

**Comment by 4x0v7** [comment]
TIL: Alpine uses Busybox uses Ash as it's shell.
Thanks for your quick reply and suggestions!
